.swiftpm/xcode/xcshareddata/xcschemes/RTSanStandaloneSwift-Package.xcscheme

```diff
@@ -15,9 +15,9 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "SwiftRealtimeSanitizer"
-               BuildableName = "SwiftRealtimeSanitizer"
-               BlueprintName = "SwiftRealtimeSanitizer"
+               BlueprintIdentifier = "RealtimeSanitizer"
```

Comment:
Does this need to be RealtimeSanitizerSwift, or RTSanStandaloneSwift or similar?

Reply:
This is referencing a buildable target, which is called `RealtimeSanitizer`. I think it is safe to ignore everything under `.swiftpm/xcode` as it is not user-facing but a convenience for the development of the library in Xcode.

Reply:
Thanks! Makes sense